index.html

```diff
@@ -1,270 +1,341 @@
+                    <h3 class="title1 ">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="Languages ">
+                        <li class="language ">Ruby on rails </li>
+                        <li class="language ">Css </li>
+                        <li class="language ">Javascript </li>
+                        <li class="language ">Html</li>
+
+                    </ul>
+                    <div class="b_div "><button class="button ">See Project</button></div>
+                </div>
+            </div>
+            <div class="cards ">
+                <div class="cardsContent ">
+                    <img src="./Images/Img Plaholder.png " alt="Store image ">
+                    <h3 class="title1">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="Languages ">
+                        <li class="language ">Ruby on rails </li>
+                        <li class="language ">Css </li>
+                        <li class="language ">Javascript </li>
+                        <li class="language ">Html</li>
+
+                    </ul>
+                    <div class="b_div "><button class="button ">See Project</button></div>
+                </div>
+            </div>
+            <div class="cards ">
+                <img src="./Images/Img Plaholder.png " alt="Store image ">
+                <h3 class="title1 ">Multi-Post Stories Gain+Glory</h3>
+                <ul class="Languages ">
+                    <li class="language ">Ruby on rails</li>
+                    <li class="language ">Css</li>
+                    <li class="language "> Javascript </li>
+                    <li class="language ">Html</li>
+                </ul>
+                <div class="b_div "><button class="button ">See Project</button></div>
+            </div>
+            <div class="cards ">
+                <img src="./Images/Img Plaholder.png " alt="Store image ">
+                <h3 class="title1 ">Multi-Post Stories Gain+Glory</h3>
+                <ul class="Languages ">
+                    <li class="language ">Ruby on rails</li>
+                    <li class="language ">Css</li>
+                    <li class="language "> Javascript</li>
+                    <li class="language ">Html</li>
+                </ul>
+                <div class="b_div "><button class="button ">See Project</button></div>
+            </div>
+
+    </section>
+    <section class="About-me " id="about">
+        <div class="aboutme-text ">
+            <h2 class="Recent ">About-me</h2>
+            <p class=" about-text1 ">I can help you build a product , feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <div class="b_div "><button class="button ">Get Resume</button></div>
+
+        </div>
+    </section>

```

Comment:
Kindly ensure that the about me section follows the design with regards to the margins and paddings.

- [ ]  The whole section seems to be pushing more towards the left side with the background image out of position and covering some of the text as shown below. Kindly consider adding a margin to the left side to push it to the right. 
- [ ] Also, don't forget to make sure the width, height, margin, padding, and all other dimensions follow the figma design in order to achieve a more responsive project that matches the actual design. 


### Your design 

![Screenshot from 2023-04-07 16-51-19](https://user-images.githubusercontent.com/65233860/230646527-c3b8d920-8a29-479d-93d3-3c96346ba5e9.png)



### Actual design 

![Screenshot from 2023-04-07 16-24-25](https://user-images.githubusercontent.com/65233860/230646399-b61457b5-6f54-47ff-ab95-375009dc3505.png)